### DISCUSSION TITLE
print tree issue

### DISCUSSION THREAD

**Discussion by dvoranag** [kayjan/bigtree#215] [Questions]
**getting an error of: **TypeError: cannot pickle 'PyCapsule' object****
when `Node ` contains additional custom and non trivial attributes 
even I invoked the `show` method with `attr_list` of name only
it trying deep copy all tree and failed 
@kayjan any advice pls?

**Comment by kayjan** [comment]
Sorry do you have example of what is the custom and non-trivial attributes and how to replicate the error you are facing? Feel free to raise an issue [here](https://github.com/kayjan/bigtree/issues)!

**Comment by kayjan** [comment]
Closing this issue due to insufficient description and inactivity
